tests/dialects/test_snowflake.py

```diff
@@ -501,7 +531,7 @@ def test_table_with_retention():
         USER_COMMENT VARCHAR(100) COMMENT 'User input',
         PROCESS_SQN NUMBER(10,0) NOT NULL,
         constraint PK_EXCLUSION primary key (ASIN)
-    ) DATA_RETENTION_TIME_IN_DAYS = 15
+    ) DATA_RETENTION_TIME_IN_DAYS=15
```

Comment:
@dmaresma I merged, but catched it after, sql in test case was valid, it can be no space near '='